app/Jobs/SendEmptyWikibaseNotificationsJob.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Notifications\EmptyWikibaseNotification;
+use App\Wiki;
+use Carbon\CarbonImmutable;
+use Illuminate\Contracts\Queue\ShouldBeUnique;
+use Illuminate\Support\Facades\Log;
+
+class SendEmptyWikibaseNotificationsJob extends Job implements ShouldBeUnique
+{
+    public function handle (): void
+    {
+        $allWikis = Wiki::all();
+        foreach ($allWikis as $wiki) {
+            try {
+                $this->sendEmptyWikibaseNotification($wiki);
+            } catch (\Exception $exception) {
+                Log::error(
+                    'Failure polling wiki '.$wiki->getAttribute('domain').' for sitestats: '.$exception->getMessage()
+                );
+            }
+        }
+    }
+
+    public function sendEmptyWikibaseNotification (Wiki $wiki): void
+    {
+        //Calculate how many days has passed since the wikibase instance was first created
+        $createdAt = $wiki->created_at;
+        $now = CarbonImmutable::now();
+        $emptyWikibaseDays = $createdAt->diffInDays($now);
+
+        $firstEdited = $wiki->first_edited;
+
+        if ($firstEdited == null && $emptyWikibaseDays >= 30) {
```

Comment:
I think this also still needs to check whether we have already sent this notification. Else we'll start spamming users heavily.

Reply:
It would be nice if the `30` would be configurable somehow, i.e. moved to `config`.